sched/wakesnoop.bt

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bpftrace
+/*
+ * latsnoop.bt   Scheduler latency tracing tool.
+ *
+ * USAGE:
+ * ./latsnoop.bt [wakeup delay threshold, us] [cpu mask] [pid]
+ *
+ * Copyright 2024 Red Hat, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ *
+ * 4-Apr-2024 	Costa Shulyupin		Initial release
```

Comment:
We're also not going to maintain a pseudo-changelog in the script headers in this repository as it's not a reliable source of information (some people update it, some don't) and to avoid debates on what changes are worth including. The Git logs should provide a complete history.

The copyright notice will of course remain.

```suggestion
```